.github/CHANGELOG.md

```diff
@@ -15,6 +15,9 @@
 
 ### Improvements
 
+* Updated Kokkos version and support to 4.3.01.
```

Comment:
Out of curiosity, why did you use `01` instead of `1` everywhere?
  
```suggestion
* Updated Kokkos version and support to 4.3.1.
```

Reply:
Because this is the version string that Kokkos uses.